lib/client.ts

```diff
@@ -89,6 +84,7 @@ class ClientManager extends EventTarget {
       refreshJwt: login.refreshJwt,
       handle: login.handle,
       did: login.did,
+      active: !!login.active,
```

Comment:
If `active` is missing then the account most likely is active.  It makes the most sense when you look at it from a compatibility perspective—this is a new field and before the field existed the accounts were assumed active.
```suggestion
      active: login.active !== false,
```